src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/LogicalExpression.java

```diff
@@ -50,69 +87,13 @@ public void traverseForNot(LogicalExpression parent) {
       }
     }
 
+    // TODO WHY ?
```

Comment:
Needs to be explained, but the idea is to replace "not" operator with inverse operations of its operands.

Like:

    !(a < b)

with

    (a >= b)

Reply:
Yes, in detail, it will traverse the tree and flip

- AND->OR, 
- OR->AND, 
- every operator to its opposite

Reply:
Yuqi Resolve: added explanation doc